### DISCUSSION TITLE
Is it possible to disable hitsounds completely without having to disable all the sound effects?

### DISCUSSION THREAD

**Discussion by Gonzalo-Bruna** [ppy/osu#22367] [Q&A] [answered]
as everybody knows, you can turn up and down the effects volume, but what if I want to completely disable the hitsounds but still hear all the other sound effects such as clicking, hovering or typing? is there any way of doing this?

**Comment by peppy** [comment, marked as answer]
It's not right now. You will need to make a skin with no hitsounds.

Hitsounds are an important element of osu! gameplay. We may add the ability to control them separately in the future depending on demand, but it's not a priority.

**Reply by Gonzalo-Bruna** [reply]
I understand. But is it a hard thing to add to the game? I've been playing osu!mania for over 8 years, and I noticed most of my friends (all of them top #50 - #300, including myself) prefer not to play with hit sounds. Probably, they've never thought of the idea that allowing sounds effects but not having hit sounds is something that could actually be added to the game, but it's very likely it's something they would love if it was added.
The problem with osu!mania is that there are so many notes that you press at the same time (specially when you are a top player and play 8+ stars songs) that hit sounds don't even sound musically right, but instead, a lot of the times they would actually distract you from the actual gameplay.
I don't think it will be much demanded for the reason I just mentioned, but please consider it (specially if it isn't a hard thing to implement, which I doubt), because adding an empty hitsound asset to the skin, probably will remove hitsounds of every other gamemode, and this is not something that I want. thanks anyway, peppy.

**Reply by peppy** [reply]
We'll probably add an extra game-wide slider for it, but it's not as trivial as you might think (especially if it is to act completely independent of the effect sound volume). It needs UX and also code structure concerns. We have other audio work to be done in similar areas (nested mixer support) which may or may not make adding such a change easier.

Keep in mind osu!mania maps with proper hitsounds (aka "keysound"? https://osu.ppy.sh/community/forums/topics/1323511?n=3) *should be played with hitsounds on*. So I fundamentally disagree with always having them off.

We also have ongoing investigation into making osu!mania hitsounds feel better in the case they aren't "keysound"ed, which may allow more users to turn hitsounds back on. We also have improvements to sound feedback which may improve the situation for some users that are distracted by the playback latency.

**Reply by Gonzalo-Bruna** [reply]
fair enough. cerrtainly, a better way to handle this would be having better hitsounds added to osu!mania. That probably would fix the problem most people have along with the latency.
I never meant disable all hitsounds by default, what I meant was to add an option to disable hitsounds if desired, which probably is most top players (at least in stable), but I understand it might require code structure refactors. hopefully, nested mixed support, will make this change easier, but as I said, probably making mania hitsounds better, will likely fix the issue.
thanks for the consideration!

**Reply by peppy** [reply]
cc/ @nekodex (more feedback on mania hitsounds being bad as they stand)
